### ISSUE TITLE
Calendar edit/create entry SEARCH function not working (to add invitees)

### ISSUE THREAD

**Issue by jobst** [salesagility/SuiteCRM#7243]
When creating or editing entry in the Calendar and you want to add invitees pressing on the search button will not find any person to add, it simply does nothing.

#### Issue
When you create a new entry in the Calendar (or edit an existing one) and want to add invitees by searching for them using the search button provided on the form nothing happens.

I increased the debug level (php/suitecrmlog), but nothing provided helped to get me further.

So I opened the Web Developer Console and received a better indication what is incorrect. There seems to be a problem with the value returned from the json_server.

```
POST https://[SUITECRM]/index.php?entryPoint=json_server

SyntaxError: JSON.parse: unexpected character at line 2 column 1 of the JSON data
method_callback https://[SUITECRM]/include/javascript/jsclass_async.js:39
    handleTransactionResponse https://[SUITECRM]/cache/include/javascript/sugar_grp1_yui.js?v=YgfFb8Znjg23F_OdH0Qfsg:109
    handleReadyState/this._poll[c.tId]< https://[SUITECRM]/include/javascript/sugar_grp1_yui.js?v=YgfFb8Znjg23F_OdH0Qfsg:109
```

If you need to debug somehow to give you more info, just tell me what to do.

#### Expected Behavior

A list of people to add to the invitee list should appear

#### Actual Behavior

Nothing happens, no error, not list

#### Possible Fix

I have never dived into the json server of SuiteCRM ... so I am out of luck here.

#### Steps to Reproduce

1. create entry in Calendar
2. press search button (bottom right) to add invitees
3. nothing happens

#### Context

People have trouble adding invitees, while it still possible to do by filling out the fields by hand (you need to copy paste not to make errors) it takes a while to go back and forth from a contact form.

#### Your Environment

* SuiteCRM Version used: 7.10.16
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Firefox 66.06
* Environment name and version (e.g. MySQL, PHP 7): 7.1
* Operating System and version (e.g Ubuntu 16.04): CentOS 7.6.1810

**Comment by cameronblaikie** [comment]
Hi there,

I could not replicate your issue, could u tell me what you have put on the invite before hitting the search button.

Thanks,

**Comment by jobst** [comment]
This has me, including all the other people using SuiteCRM, baffled!

We had this problem for many, many weeks, all of us - we also could not figure out when it started.
We tried one letter, a known first name, last name, false names - you name it we tried.
Nothing ever came up.

We tried today and it worked. Note I have not done any updates to the SuiteCRM install. I upgraded it to 7.10.16 when we discovered the problem - although at the time it did not help.

No idea whether something changed in Firefox, we have automatic updates enabled.
No idea what made it work, I have so much work on that I really had not time to put any effort into this.

Feel like a complete dill.
